Gemfile.lock

```diff
@@ -762,7 +762,7 @@ DEPENDENCIES
   wkhtmltopdf-binary
 
 RUBY VERSION
-   ruby 2.1.5p273
+  ruby 2.1.5p273
 
 BUNDLED WITH
-   1.15.1
+  1.15.1
```

Comment:
yep when you run `bundle install` it checks for changes in `Gemfile` (where we declare our dependencies) and generates a `Gemfile.lock` to fix exactly which versions we are using.

if you don't need any change in `Gemfile` just run `git checkout Gemfile.lock` and run `bundle install` again. Please ping me if get stuck at some point 💪